<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolDialog</class>
 <widget class="QDialog" name="ToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabSearch">
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>查找</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>查找目标：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SearchCount">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>55</y>
       <width>111</width>
       <height>76</height>
      </rect>
     </property>
     <property name="text">
      <string>计数</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_SearchReverse">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>反向查找</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabReplace">
    <attribute name="title">
     <string>替换</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_Replace">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>55</y>
       <width>111</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>替换</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>25</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>替换目标：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>55</y>
       <width>53</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>替换为：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditReplaceTo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>55</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ReplaceAll">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>替换所有</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>窗口透明度</string>
    </property>
    <widget class="QRadioButton" name="radioButton_Close">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>95</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_Always">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>始终</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>60</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>80</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>47</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEditTarget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>161</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_Cancal">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>43</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_SearchNext">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>查找下一个</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
